markdown/container.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react'
+import { loadMarkdown, saveMarkdown } from './service'
+import { View } from './view'
+import { ContainerProps } from '../types'
+
+export const Container: React.FC<ContainerProps> = (props) => {
```

Comment:
Props should be typed of course, but looks like a reasonable way of passing data

Reply:
They are typed :) The generic passed to `React.FC` (i.e. `ContainerProps`) define the types of the props. (I chose to extract them to a separate file here to encourage using the same API for all top-level containers - that would make it easier to potentially extract panes into separate apps, for example.)